---
title: LOCALDB_ERROR_UNKNOWN_INSTANCE |Microsoft Docs
ms.custom: ''
ms.date: 03/03/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: 67bca4f1-915d-46d4-9293-3742c16930a1
author: stevestein
ms.author: sstein
ms.openlocfilehash: 7e624dfd3c00f97debd9b309b5b534a262b7f7b2
ms.sourcegitcommit: da88320c474c1c9124574f90d549c50ee3387b4c
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 07/01/2020
ms.locfileid: "85641436"
---
# <a name="localdb_error_unknown_instance"></a>LOCALDB_ERROR_UNKNOWN_INSTANCE
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
    
## <a name="details"></a>詳細  
  
|||  
|-|-|  
|製品名|SQL Server|  
|イベント ID|263|  
|イベント ソース|SQL Server Local Database Runtime 12.0|  
|コンポーネント|Local Database Runtime API|  
|メッセージ テキスト|指定したローカル データベース インスタンスは存在しません。|  
  
## <a name="explanation"></a>説明  
 指定したインスタンスは存在しません。  
  
## <a name="user-action"></a>ユーザーの操作  
 指定したインスタンス名が正しいかどうかを確認してください。  
  
  
